subjects/getascii/README.md

```diff
@@ -0,0 +1,28 @@
+## GetAscii
+
+### Instructions
+
+Write a program that takes a character as an arguments, and displays its ascii followed by a newline.
```

Comment:
Fix the typo: argument instead of arguments, and ASCII should be in uppercase

Reply:
it's fixed